src/inverseRobot.js

```diff
@@ -7,7 +7,22 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
+  const values = Object.values(robot);
+
+  for (let i = 0; i < values.length; i++) {
+    if (values
+      .slice(i + 1, values.length)
+      .includes(values[i])) {
+      return null;
+    }
+  }
+
+  for (const key in robot) {
```

Comment:
```suggestion
  for (const key in robot) {
     if (robot[key] in result) {
      return null;
    }
```
you can add this check right on `for...in` loop